app/controllers/admin/labels_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Admin::LabelsController < Admin::BaseController
     def index
-        @labels = ActsAsTaggableOn::Tag.joins(:taggings).where(taggings: { context: 'labels' }).distinct.most_used(100)
+        @labels = ActsAsTaggableOn::Tag.joins(:taggings).where(taggings: { context: 'labels' }).distinct.sort_by {|tags| -tags.name}
```

Comment:
I'm not sure a - is the neatest way to reverse this sort?

Reply:
Yes, we can use `order` which is a bit easier to read:
```suggestion
        @labels = ActsAsTaggableOn::Tag.joins(:taggings).where(taggings: { context: 'labels' }).distinct.order(:name)
```

It defaults to ascending order, but you can also do `.order(name: :desc)` if you want reverse alphabetical order